WORKSPACE

```diff
@@ -11,3 +12,5 @@ new_local_repository(
     build_file = "@com_justbuchanan_rules_qt//:qt.BUILD",
     path = local_qt_path(),
 )
+
+register_qt_toolchains()
```

Comment:
My idea regarding qt_configure and register_qt_toolchains was that in the future both are merged in one single call and you only need to call register_qt_toolchain. One option to do that is with template files, similar to what is done in rules_go https://github.com/bazelbuild/rules_go/blob/4cd45a2ac59bd00ba54d23ebbdb7e5e2aed69007/go/private/sdk.bzl

Reply:
Sounds good, I agree it would be best to put everything in just a single function that needs to be called from WORKSPACE. I'll look at refactoring this to do so.